src/js/helpers.js

```diff
@@ -458,7 +458,14 @@ TENANT_TOKEN="${token}" && \\`
 mender setup \\
   --device-type $DEVICE_TYPE \\
 ${connectionInstructions} && \\
-systemctl restart mender-client'
+systemctl restart mender-client && \\
+(cat > /etc/mender/mender-shell.conf << EOF
+{
+  "ServerURL": "https://hosted.mender.io",
```

Comment:
This might be `"ServerURL": "${document.location.origin}"` - the file itself is likely not problematic for on-prem OS installations, but I guess on-prem enterprise could have deviceconnect installed as well and could then reuse the snippet...

Reply:
Updated, thanks!